2023 02 05 madelein new logging state changes

madeleinvillegas opened the pull request:
Removes multiple calls to CreatePair

madeleinvillegas committed:
logging state changes in JSON

madeleinvillegas committed:
commit to get an apk generated

madeleinvillegas committed:
Adding visual log states

madeleinvillegas committed:
name changes

madeleinvillegas committed:
Merge remote-tracking branch 'origin/development' into 2023-02-05-Madelein-NEW-Logging-State-Changes

# Conflicts:
#	app/src/main/java/com/example/iris/login1/GalleryActivity.java

madeleinvillegas committed:
remove CreatePair

JackMostow reviewed (dismissed):
Some odd identifiers predated your changes, so we're probably stuck with them, but do you happen to know the reasons for their names, e.g. "genderlay" or "activity_gal"?

Could spaces in "Record Thread Start" or "Logo ACTION UP" cause problems in parsing logs? logHandler.log("recording_thread", "Record Thread Start");
logHandler.log("animation_start", "Logo ACTION UP");

After awhile I stopped checking every logHandler.log call to see if it made sense.  Calls with variables or constants as arguments shouldn't break.
Only calls should break only if they have expressions as arguments, like this call:
logHandler.log("audio_start", getResources().getResourceEntryName(id));

JackMostow committed:
Merge branch 'development' into 2023-02-05-Madelein-NEW-Logging-State-Changes

JackMostow reviewed (dismissed):
All changes simply eliminated Pair.create except for adding String.valueOf in logHandler.log("visual_state_visible", String.valueOf(slideRoboFingerIcon.getId()));

JackMostow commented:
@madeleinvillegas - After I resolved all 54 conflicts, I got:
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

Can you check the compiler error output?  I don't know where to find it.

JackMostow committed:
Update GalleryActivity.java

Jack deleted two < apparently responsible for compiler errors:

/home/runner/work/RTFace_Login/RTFace_Login/app/src/main/java/com/example/iris/login1/GalleryActivity.java:2027: error: > expected

<                        logHandler.log("visual_state_gone", "icongall");
> Task :app:compileDebugJavaWithJavac FAILED
                                       ^
/home/runner/work/RTFace_Login/RTFace_Login/app/src/main/java/com/example/iris/login1/GalleryActivity.java:2035: error: > expected

<                        logHandler.log("visual_state_visible", "activity_gal");

JackMostow reviewed (dismissed):
Looks OK to me (= Jack) including deleting two extraneous < by editing code directly in GitHub

JackMostow committed:
Update GalleryActivity.java

Added missing // after my 2 comments

JackMostow committed:
Update GalleryActivity.java

deleted "nt" apparently typed by accident

JackMostow reviewed (approved):
I hope the typo fixes were the only changes.

JackMostow commented:
@madeleinvillegas - The build finally succeeded so Jack proceeded with the merge to satisfy the pull request, but I don't want to delete your branch without your blessing.
Also, where can I find the resulting apk?
